db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1,2 @@
+-- sql table for peeps data
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, peep VARCHAR(150));
```

Comment:
Further down you have a thing that inserts the time into the peeps table - is there a time column? Or is it working in a different way?

Reply:
Yeah I forgot to update that to be honest...